tests/test_metalog.py

```diff
@@ -12,6 +12,7 @@
 from pathlib import Path
 
 sys.path.append(str(Path(__file__).parent.parent))
+import pycheribuild.check_version  # noqa
```

Comment:
We probably don't need this for the tests, but adding it to the main entry point seems reasonable.